### PR TITLE
Home-3-sections-of-pictures

### PR DESCRIPTION
**Pull request by danielius89**

### CODE REVIEW

**Comment by danielius89** [comment]
Home-3-sections-of-pictures

**Comment by danielius89** [comment]
feat: created HTML structure for Destinations section.
Waiting for SCSS team to prepare Bootstrap to continue my task.

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,25 @@
...
+            <div class="col-4"> place for image</div>
+            <div class="col-8"> place for text</div>
+        </div>
+        <div class="row">
+            <div class="col-8"> place for image</div>
+            <div class="col-4"> place for text</div>
+        </div>
+    </div>
 </body>
 </html>
```
Comment (ZilvinasVidmantas) [inline public/index.html:32]: missing EOF

Comment (danielius89) [reply public/index.html]: added missing EOF

**Review by danielius89 — COMMENTED** [review]
bugfix: added missing EOT, indent size changed to 2 spaces

**Review by gediminas-kubilius — APPROVED** [review]
nu super (y)

**Review by ZilvinasVidmantas — APPROVED** [review]

**Pull request merged**